test/web-platform-tests/to-run.yaml

```diff
@@ -577,48 +686,49 @@ DIR: html/canvas/element/drawing-images-to-the-canvas
 
 ---
 
-DIR: html/dom/documents/dom-tree-accessors
-
-Document.currentScript.html: [timeout, Unknown]
-document.getElementsByName/document.getElementsByName-namespace-xhtml.xhtml: [fail, Needs MathML support]
-document.getElementsByName/document.getElementsByName-namespace.html: [fail, Needs MathML support]
-nameditem-01.html: [fail, Unknown]
-nameditem-02.html: [fail, Unknown]
-nameditem-04.html: [fail, Unknown]
-nameditem-05.html: [fail, Unknown]
-nameditem-06.html: [fail, Unknown]
-nameditem-07.html: [fail, Unknown]
-nameditem-08.html: [fail, Unknown]
-nameditem-names.html: [fail, Unknown]
-
----
-
-DIR: html/dom/documents/resource-metadata-management
-
-document-lastModified-01.html: [fail, Unknown]
-document-lastModified.html: [fail, Unknown]
-
----
-
-DIR: html/dom/elements/elements-in-the-dom
-
----
-
-DIR: html/dom/elements/global-attributes
-
-dataset-prototype.html: [fail, Tests Object.prototype which jsdom has trouble with due to VM globals]
-dir-bdi-script.html: [fail, Unknown]
-dir-slots-directionality.tentative.html: [fail, Unknown]
-the-lang-attribute-001.html: [fail, Unknown]
-the-lang-attribute-002.html: [fail, Unknown]
-the-lang-attribute-003.html: [fail, Unknown]
-the-lang-attribute-004.html: [fail, Unknown]
-the-lang-attribute-005.html: [fail, Unknown]
-the-lang-attribute-006.html: [fail, Unknown]
-the-lang-attribute-007.html: [fail, Unknown]
-the-lang-attribute-008.html: [fail, Unknown]
-the-lang-attribute-009.html: [fail, Unknown]
-the-lang-attribute-010.html: [fail, Unknown]
+DIR: html/dom
```

Comment:
this whole section (and subdirs) got refactored due to the `aria-` files getting moved over from `dom/nodes`; to keep testing those two I had reorganize things this way (maybe there's a better way?)

Reply:
I think this is a weakness of our current setup, so there's not much we can do. Thanks for doing all that extra work.